core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -298,6 +299,11 @@ public SystemSessionProperties(
                         "Scale out writers based on throughput (use minimum necessary)",
                         featuresConfig.isScaleWriters(),
                         false),
+                integerProperty(
+                        MAX_WRITER_NODES_COUNT,
```

Comment:
Maybe `MAX_WRITER_NODES_COUNT` -> `MAX_WRITER_TASKS_COUNT`? Then we can use the same property later on in FTE mode. Since in FTE we can have multiple tasks per node